package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "dex-price-estimator",
+  "version": "0.0.1",
+  "description": "A service estimating feasible sell prices for Gnosis Protocol",
+  "main": "dist/index.js",
+  "scripts": {
+    "prebuild": "tslint -c tslint.json -p tsconfig.json --fix",
+    "build": "tsc",
+    "prestart": "npm run build",
+    "start": "node .",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [
+    "Gnosis",
+    "Protocol",
+    "price",
+    "estimation",
+    "dfusion"
+  ],
+  "author": "Gnosis",
+  "license": "ISC",
```

Comment:
I would leave this out for now until we decide what the license should be.

Reply:
MIT as telegram-bot, right?

Reply:
Leaving it out for now. Why is telegram using MIT over Apache?

Reply:
From what I've read recently, a good license is MIT + Apache2 dual license. The advantage of MIT is that it is GPL2 compatible (which Apache is not).

Reply:
We checked at the beginning, and we were told MIT. Double check in #lega-requests, I'm not able to answer this :)